packages/style-utilities/src/utilities/icons.ts

```diff
@@ -219,7 +219,7 @@ export function setIconOptions(options: Partial<IIconOptions>): void {
 }
 
 let _missingIcons: string[] = [];
-let _missingIconsTimer: number | undefined = undefined;
+let _missingIconsTimer: ReturnType<typeof setTimeout> | undefined = undefined;
```

Comment:
this wont be `number` in node js, thus this is the safe approach to have uniform data types across environments.

- in browser it will be `number`
- in nodejs `NodeJS.Time`